stdlib/3/email/headerregistry.pyi

```diff
@@ -1,6 +1,6 @@
 # Stubs for email.headerregistry (Python 3.4)
 
-import datetime as dt
+from datetime import datetime as dt
```

Comment:
This will make the stub re-export `datetime.datetime` as `dt`, which isn't right. Can you change it to just `from datetime import datetime`?